astro/src/content/docs/lifecycle/authenticate-users/single-sign-on.mdx

```diff
@@ -168,417 +156,137 @@ npm install
+The session length for this application is 60 seconds; the `maxAge` value is in milliseconds. When the node application's session expires, it will redirect the end user to FusionAuth. If the single sign-on session has not expired, the user will be transparently redirected back. If it has expired, the user must re-authenticate.
 
 ### The `www` script
 
 The next step is to create a script which starts up express. Place the contents of this file in `bin/www`.
 
-```javascript title="bin/www"
-#!/usr/bin/env node
-
-/**
- * Module dependencies.
- */
-
-var app = require('../app');
-var debug = require('debug')('fusionauth-node-example:server');
-var http = require('http');
-
-/**
- * Get port from environment and store in Express.
- */
-
-var port = normalizePort(process.env.PORT || '3000');
-app.set('port', port);
-
-/**
- * Create HTTP server.
- */
-
-var server = http.createServer(app);
-
-/**
- * Listen on provided port, on all network interfaces.
- */
-
-server.listen(port);
-server.on('error', onError);
-server.on('listening', onListening);
-
-/**
- * Normalize a port into a number, string, or false.
- */
-
-function normalizePort(val) {
-  var port = parseInt(val, 10);
-
-  if (isNaN(port)) {
-    // named pipe
-    return val;
-  }
-
-  if (port >= 0) {
-    // port number
-    return port;
-  }
-
-  return false;
-}
-
-/**
- * Event listener for HTTP server "error" event.
- */
-
-function onError(error) {
-  if (error.syscall !== 'listen') {
-    throw error;
-  }
-
-  var bind = typeof port === 'string'
-    ? 'Pipe ' + port
-    : 'Port ' + port;
-
-  // handle specific listen errors with friendly messages
-  switch (error.code) {
-    case 'EACCES':
-      console.error(bind + ' requires elevated privileges');
-      process.exit(1);
-      break;
-    case 'EADDRINUSE':
-      console.error(bind + ' is already in use');
-      process.exit(1);
-      break;
-    default:
-      throw error;
-  }
-}
-
-/**
- * Event listener for HTTP server "listening" event.
- */
-
-function onListening() {
-  var addr = server.address();
-  var bind = typeof addr === 'string'
-    ? 'pipe ' + addr
-    : 'port ' + addr.port;
-  debug('Listening on ' + bind);
-}
-```
+<RemoteCode
+  url={frontmatter.codeRoot + "/pied-piper/bin/www"}
+  lang="javascript" 
+  title="bin/www" />
 
 This script is what running `npm start` actually executes.
 
 This code isn't that interesting with regards to single sign-on but is included for completeness. 
 
-It looks for a port from the environment or uses `3000` as the default. It also registers some error handling code. Then it starts up a server listening on that port, based on configuration from `app.js`.
+It looks for a port from the environment or uses `3000` as the default. It also registers some error handling code. Then it starts up a server listening on that port, based on configuration from `app.js`.'
+
+### The .env File
+
+Here's the `.env` file, which should be placed at `.env`:
+
+<RemoteCode
```

Comment:
do we have a language that would highlight .env scripts better?

Using javascript sure looks nicer when I try it
<img width="1447" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/139158618/b924f694-8a61-4ec3-af0a-401fd9a0146b">

Reply:
Looks like we should use "properties": https://prismjs.com/#supported-languages has the list.